.github/workflows/test.yml

```diff
@@ -248,12 +246,13 @@ jobs:
           go-version: ${{ matrix.go-version }}
           cache: true
           check-latest: true
-      - name: "Cross"
+      - name: "build"
```

Comment:
Better display. This is not "cross-building" it is building for different versions of go.